client/js/widgets/scoreboard.js

```diff
@@ -48,6 +48,54 @@ class Scoreboard extends Widget {
     return p;
   }
 
+  async click(mode='respect') {
+    if(!await super.click(mode)) {
+      const scoreProperty = this.get('scoreProperty');
+      const seats = this.getIncludedSeats();
```

Comment:
I think that this is the point at which you should simply abort the edit if the result is a non-array object, since I don't really see how to make sense of editing team scores.

Reply:
Fixed as proposed.